docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: '3.7'
+
+services:
+  db:
+    image: "postgres"
+    environment:
+      - "POSTGRES_HOST_AUTH_METHOD=trust"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
```

Comment:
```ports:
  - 5432: 5432```

exposes port to local machine... that you you can hit it from localhost:5432

Reply:
handled by @shinjonathan